pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/cli/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: layout.pug
+navigationTitle: Beeline CLI
+excerpt: 
+title: Using the Beeline CLI in Hive Metastore
+menuWeight: 50
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+enterprise: true
+---
+
+Users can interact with the {{ model.techName }} via the Beeline CLI.
+
+# <a name="Using MySQL Database"></a> Using MySQL Database
+
+- Please make sure the `Hive Metastore` & `MySQL` services are deployed and successfully runnning.
+
+- Find the load balanced address for mysql. You'll need this to setup the connection with Beeline. You can find the LB address in the DC/OS UI under the `Services > mysql > Endpoints` tab. It should look similar to:
```

Comment:
Has anyone tested this procedure? I logged in to a cluster and could not load MySQL from the Catalog, let alone find its Endpoints tab. I think this section may be out of date.

Reply:
Hi Sarah, yup we made sure to test this. Took the following screenshot in soak113:

<img width="963" alt="Screen Shot 2019-10-28 at 5 25 17 PM" src="https://user-images.githubusercontent.com/4225547/67727815-14c83d80-f9a8-11e9-95dc-2ae2e6fed062.png">

<img width="1004" alt="Screen Shot 2019-10-28 at 5 24 51 PM" src="https://user-images.githubusercontent.com/4225547/67727849-33c6cf80-f9a8-11e9-945a-2113cf7e3db6.png">

But maybe we should add a link here to point back to [this set of instructions] for installing mysql (https://github.com/mesosphere/dcos-docs-site/pull/2582/files/540ca61c4501ed7b529b02d51618081ce2a381e1?short_path=6e13cb2#diff-4e79d80993ee256d01f8bdf89cf9b2deR26)

Reply:
No, I just wanted to verify that this step works. Thanks.